doc/versions.json

```diff
@@ -17,16 +17,16 @@
   ],
   "COMPONENTS_BY_VERSION": {
     "latest": {
-      "ncs": "1.4.99",
-      "nrfxlib": "1.4.99",
+      "ncs": "1.5.0",
+      "nrfxlib": "1.5.0",
       "zephyr": "2.4.99",
-      "mcuboot": "1.6.99"
+      "mcuboot": "1.7.99"
```

Comment:
If mcuboot is at v1.7.99 now, this also needs to be updated in nrf/doc/mcuboot/conf.py
Actually, that should have been done during the mcuboot upmerge that put the version at 1.7.99 - @carlescufi , is that missing in some checklist for upmerges?